src/components/room-invitation-creation-modal.js

```diff
@@ -98,7 +98,7 @@ function RoomInvitationCreationModal({ isOpen, onOk, onCancel, roomId }) {
           rules={emailsValidationRules}
           normalize={normalizeEmails}
           >
-          <TextArea onChange={handleOnEmailsChange} autoSize={{ minRows: 3, maxRows: 12 }} />
+          <NeverScrollingTextArea onChange={handleOnEmailsChange} minRows={3} />
```

Comment:
This would have been the only remaining occurrence of using the AntD `TextArea` directly, so I decided to also use our auto-expanding wrapper.